doc/tutorials/building_uos_from_clearlinux.rst

```diff
@@ -0,0 +1,160 @@
+*************************************
+
+In order to build out the image of UOS, 
+please follow these steps in Clear Linux native:
+
+#. In Clear Linxu native, please install ``ister`` at first, 
+   ``ister`` is a template based installer for linux, 
+   which is inclued in the bundle ``os-installer`` in Clear Linux.
+   For more information about ``ister``, 
+   please visit https://github.com/bryteise/ister.
+
+   .. code-block:: none
+
+      # sudo swupd bundle-add os-installer
+   
+#. After installation is complete, use ``ister.py`` to 
+   generate the image for UOS with the configuration in 
+   ``uos-image.json``:
+
+   .. code-block:: none
+   
+      # cd ~   
+      # sudo ister.py -t uos-image.json
+      
+   An example of the configuration file ``uos-image.json`` 
+   is shown as below:
+
+   .. code-block:: none
+   
+      {
+          "DestinationType" : "virtual",
+          "PartitionLayout" : [ { "disk" : "uos.img",
+                                  "partition" : 1,
+                                  "size" : "512M",
+                                  "type" : "EFI" },
+                                { "disk" : "uos.img",
+                                  "partition" : 2,
+                                  "size" : "32M",
+                                  "type" : "swap" },
+                                { "disk" : "uos.img",
+                                  "partition" : 3,
+                                  "size" : "8G",
+                                  "type" : "linux" } ],
+          "FilesystemTypes" : [ { "disk" : "uos.img",
+                                  "partition" : 1,
+                                  "type" : "vfat" },
+                                { "disk" : "uos.img",
+                                  "partition" : 2,
+                                  "type" : "swap" },
+                                { "disk" : "uos.img",
+                                  "partition" : 3,
+                                  "type" : "ext4" } ],
+          "PartitionMountPoints" : [ { "disk" : "uos.img",
+                                       "partition" : 1,
+                                       "mount" : "/boot" },
+                                     { "disk" : "uos.img",
+                                       "partition" : 3,
+                                       "mount" : "/" } ],
+          "Version": "latest",
+          "Bundles": ["bootloader",
+                      "editors",
+                      "kernel-iot-lts2018",
+                      "network-basic",
+                      "os-core-update",
+                      "os-core",
+                      "openssh-server",
+                      "software-defined-cockpit",
+                      "sysadmin-basic"]
+       }
+
+   .. note::
+      To generate the image with a specified version, 
+      please modify with the ``"Version"`` argument, 
+      and we can set ``"Version": 26550`` for example.
```

Comment:
I would add a note here to make it explicit that `"latest"` *must* be in quotes while if you use a version number it must *not* be in quotes.